### PR TITLE
Remove Kopano specific defaults and naming for white label rename

### PR DESCRIPTION
**Pull request by longsleep**

### CODE REVIEW

**Comment by longsleep** [comment]
@fbartels It would be great if you could take care of the README in a similar fashion when you have the time.

File: docs/example-template.ldif [hunk 1]
```diff
@@ -1,87 +1,76 @@
...
 {{end}}
 
 {{with .}}
 {{$cn := "Mary Poppins" -}}
-dn: cn={{$cn}},ou=contacts,ou={{.Company}},{{.BaseDN}}
+dn: cn={{$cn}},ou=contacts,o={{.Company}},{{.BaseDN}}
 objectClass: inetOrgPerson
 objectClass: top
-objectClass: kopano-contact
-uidNumber: {{AutoIncrement}}
```
Comment (fbartels) [inline docs/example-template.ldif:81]: deleted by accident?

File: server/handler/ldif/index.go [hunk 2]
```diff
@@ -22,25 +22,32 @@ var indexAttributes = map[string]string{
 	"gidNumber":    "eq",
 	"mail":         "eq",
 	"memberUid":    "eq",
+	"o":            "eq",
 	"ou":           "eq",
 	"uid":          "eq",
 	"uidNumber":    "eq",
 	"uniqueMember": "eq",
 
-	"kopanoAccount":       "eq",
```
Comment (fbartels) [inline server/handler/ldif/index.go:30]: does these indexes then need to be re-added in a Kopano specific build?

File: scripts/10-main.ldif.in [hunk 3]
```diff
@@ -4,28 +4,27 @@
 dn: {{.BaseDN}}
 objectClass: top
 
-dn: ou={{.Company}},{{.BaseDN}}
-objectClass: organizationalUnit
+dn: o={{.Company}},{{.BaseDN}}
```
Comment (fbartels) [inline scripts/10-main.ldif.in:7]: Not totally sure about the organizationalUnit to organization change. The only real argument I have against it is however that you cannot make it a kopano-company if its not an organizationalunit.

https://github.com/Kopano-dev/kopano-core/blob/a16ffc825e4e7a6d6da81fa17cd52a59f41f3038/installer/ldap/kopano.schema#L247-L260

(but on the other hand the Kopano ldap schema definitions are not too relevant in the scope of this project)

**Review by fbartels — CHANGES_REQUESTED** [review]
I think we need to have a short chat about the organization vs. organizationunit change that this pr brings.

File: docs/example-template.ldif (hunk 1 above)
Comment (longsleep) [reply docs/example-template.ldif:81]: No, deleted intentionally.

File: server/handler/ldif/index.go (hunk 2 above)
Comment (longsleep) [reply server/handler/ldif/index.go:30]: Yes

File: scripts/10-main.ldif.in (hunk 3 above)
Comment (longsleep) [reply scripts/10-main.ldif.in:7]: Mhm i see - i am already considering keeping the `ou` thing - but i also want to make it possible to differenciate between the different organizationalUnits.

**Review by fbartels — APPROVED** [review]

**Pull request merged**